src/main/cluster.ts

```diff
@@ -70,6 +70,7 @@ export class Cluster implements ClusterModel {
   @observable features: FeatureStatusMap = {};
   @observable allowedNamespaces: string[] = [];
   @observable allowedResources: string[] = [];
+  @observable accessibleNamespaces?: string[];
```

Comment:
Maybe we can set initial value as in properties above like
`@observable accessibleNamespaces: string[] = [];`?